Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -126,39 +128,34 @@ std::string VolumeGC::GetInternalName(const Partition& partition) const
 std::map<Language, std::string> VolumeGC::GetShortNames() const
 {
   LoadBannerFile();
```

Comment:
Can this call be removed?

Reply:
Yes. It seems like I just forgot to remove it.